### ISSUE TITLE
Home page prompts user to install Flash when in Tor Browser's High Security mode

### ISSUE THREAD

**Issue by JeremyRand** [bisq-network/bisq-website#40]
When I visit https://bisq.network/ in Tor Browser's High Security mode (which implies that JS is disabled), the Vimeo video isn't playable (which is mildly bad), and Vimeo prompts me to install Flash (which is much worse).

I'm not sure what the right way to fix this is (ideally if you can host an MP4 or WEBM file directly without using a player, then Tor Browser will let the user play it without enabling JS), but I figure it's worth reporting.

**Comment by JeremyRand** [comment]
The message from Vimeo is: "What’s going on here?  Some of your technology may be out of date, which means this video won’t play properly. Please upgrade your browser or install Flash."

Note that the above message only appears for a subset of Tor circuits.  For the remaining Tor circuits, Vimeo instead says "You have been temporarily blocked.  Pardon the inconvenience, but our servers have detected a high number of errors from your connection. To continue, please verify that you are a human:"  (No CAPTCHA is displayed.)

**Comment by cbeams** [comment]
Thanks, @JeremyRand. Do you know what Tor Browser's High Security mode does with YouTube embedded videos? We're planning to move the explainer video (and everything else) from our Vimeo account to our new YouTube channel as part of https://github.com/bisq-network/dao/issues/22.

**Comment by JeremyRand** [comment]
> Thanks, @JeremyRand. Do you know what Tor Browser's High Security mode does with YouTube embedded videos? We're planning to move the explainer video (and everything else) from our Vimeo account to our new YouTube channel as part of bisq-network/dao#22.

@cbeams the embedded YouTube video that's now on https://bisq.network/ displays the following error: "An error occurred.  Try watching this video on www.youtube.com, or enable JavaScript if it is disabled in your browser."

This is certainly better than prompting the user to install Flash, although it would be even better if there were a way that you guys could directly embed a video file without requiring JavaScript.

**Comment by m52go** [comment]
The site has been thoroughly overhauled since this issue was created. The video in question was removed.

Generally speaking I think Flash is all but buried now, but to be sure, I accessed [a page with an embedded YouTube video](https://bisq.network/blog/bisq-dao-for-bitcoin-maximalists/) on the Tor browser in high-security mode and there was no prompt to install Flash. Vimeo videos are no longer used on the site.

Closing as fixed/no longer an issue.
